generators/Exercises/Connect.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Generators.Input;
+using Generators.Output;
+
+namespace Generators.Exercises
+{
+    public class Connect : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                canonicalDataCase.UseVariablesForConstructorParameters = true;
+                canonicalDataCase.SetConstructorInputParameters("board");
+
+                //remove whitespace
+                var trimBoard = new List<string>();
```

Comment:
Why trim the white space? The original implementation also doesn't trim it and this makes it look better IMHO. Could you remove the trimming?

Reply:
If I don't trim it the tests where failing because I think it's looping through every character and taking the spaces as an empty space. I could be wrong, I'll look into it.